_posts/2023-11-09-in-progress-feature-on-openshift.adoc

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: 'Trying Out An In-progress Feature for WildFly on OpenShift'
```

Comment:
Looks good! Would be good to add wildfly in the filename too, in-progress-wildfly-feature-on-openshift.adoc

Reply:
Fixed!